tests/test_prompt_registry_mongo.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+ip_address=$(hostname -I | awk '{print $1}')
+
+export MONGO_HOST=${ip_address}
+export MONGO_PORT=27017
+export DB_NAME=${DB_NAME:-"Prompts"}
+export COLLECTION_NAME=${COLLECTION_NAME:-"test"}
+
+function build_docker_images() {
+    cd $WORKPATH
+    echo $(pwd)
+    docker run -d -p 27017:27017 --name=mongo mongo:latest
+
+    docker build -t opea/promptregistry-mongo-server:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/prompt_registry/mongo/docker/Dockerfile .
+}
+
+function start_service() {
+
+    docker run -d --name="promptregistry-mongo-server" -p 6012:6012 -p 6013:6013 -p 6014:6014 -e http_proxy=$http_proxy -e https_proxy=$https_proxy -e no_proxy=$no_proxy -e MONGO_HOST=${MONGO_HOST} -e MONGO_PORT=${MONGO_PORT} -e DB_NAME=${DB_NAME} -e COLLECTION_NAME=${COLLECTION_NAME} opea/promptregistry-mongo-server:latest
+
+    sleep 10s
+}
+
+function validate_microservice() {
+    result=$(curl -X 'POST' \
+  http://$ip_address:6012/v1/prompt/create \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "prompt_text": "test prompt", "user": "test"
+}')
+    echo $result
+    if [[ ${#result} -eq 26 ]]; then
+        echo "Correct result."
+    else
+        echo "Incorrect result."
+        exit 1
+    fi
+
+}
+
+function stop_docker() {
+    cid=$(docker ps -aq --filter "name=mongo*")
```

Comment:
Please also clean up `promptregistry-mongo-server`, so suggest to use `test-comps-` as container prefix, so here you can use ` cid=$(docker ps -aq --filter "name=test-comps-*")` to remove all the test images.

Reply:
Done changes.